### PR TITLE
Update statistics for prod67 release

### PR DESCRIPTION
**Pull request by sbesson**
This release includes:

- another set of 1000 antibodies from the HPA project
- a new study `idr0062`

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

**Comment by sbesson** [comment]
Staged at https://snoopycrimecop.github.io/idr.openmicroscopy.org/studies.html

File: _data/studies.tsv [hunk 1]
```diff
@@ -47,7 +47,7 @@ idr0038-held-kidneylightsheet	experimentC	prod44	353	3	0	3	2	0	12	236872	1.70676
 idr0040-aymoz-singlecell	experimentA	prod47	401	76	0	76		0	780	86116	0.7258287638	725828763846	88205	8.228884574	2048 x 2048 x 1 x 5 x 21.5
 idr0041-cai-mitoticatlas	experimentA	prod47	404	276	0	276		0	60480	3749760	0.5014639821	501463982132	181442	2.763770142	256 x 256 x 31 x 2 x 1
 idr0042-nirschl-wsideeplearning	experimentA	prod47	402	2	0	2		0	2299	6897	0.00037565707	375657070	6898	0.05445883879	250 x 250 x 1 x 3 x 1
-idr0043-uhlen-humanproteinatlas	experimentA	prod52	501	4000	0	4000		0	1965418	5896254	53.46299701	53462997006816	1477697	36.17994556	3000 x 3000 x 1 x 3 x 1
+idr0043-uhlen-humanproteinatlas	experimentA	prod52	501	5000	0	5000		0	2452054	7356162	66.1975	66197500000000	2452818	26.98834565	3000 x 3000 x 1 x 3 x 1
```
Comment (manics) [inline _data/studies.tsv]: Looks like quite a large drop in `avg. size (MB)` from `36.17994556` to `26.98834565`, but assuming that's correct....

Comment (sbesson) [reply _data/studies.tsv]: Agreed the average size had increased in d695f7af97679315a93a3b8b3e1d32d26cb74027 but I think it was the result of a miscalculation. I can print out the numbers for the bytes/number of files later here.

**Pull request merged**